docs/pages/modules/get-started.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Get Started
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+import { Terminal } from '~/ui/components/Snippet';
+
+## Create a new module
+
+To create a new Expo module from scratch, just run the `create-expo-module` script as below. The script will ask you a few questions and then generate the native Expo module along with the example app for iOS and Android that uses your new module.
+
+<Terminal cmd={[`$ npx create-expo-module`]} />
+
+## Set up a library as an Expo module
+
+You may want to use the Expo Modules API in existing React Native libraries, for example with [AppDelegate Subscribers](./appdelegate-subscribers.mdx) you can hook into `AppDelegate` methods without requiring developers to copy any code over to their own `AppDelegate`. This is particularly useful to add seamless support for Expo managed projects to a library. The following steps will set up your existing React Native library to have access to the Expo Modules API.
+
+Create the [module config](./module-config.mdx) **expo-module.config.json** file just near your **package.json** and start from the empty object `{}` in there. We will fill it in later to enable specific features.
+The presence of the module config is enough for Expo Autolinking to recognize it as an Expo module and automatically link your native code.
+
+### Add the `expo-modules-core` native dependency
+
+Add `expo-modules-core` as a dependency in your podspec and **build.gradle** files.<br/>
+
+<CodeBlocksTable tabs={["*.podspec", "build.gradle"]}>
+
+```ruby
+# ...
+Pod::Spec.new do |s|
+  # ...
+  s.dependency 'ExpoModulesCore'
+end
+```
+
+```groovy
+// ...
+dependencies {
+  // ...
+  implementation project(':expo-modules-core')
+}
+```
+
+</CodeBlocksTable>
+
+### Add Expo packages to dependencies
+
+Add `expo` package as a peer dependency in your **package.json** — we recommend using `*` as a version range so as not to cause any duplicated packages in user's **node_modules** folder. Your library also needs to depend on `expo-modules-core` but only as a dev dependency — it's already provided in the projects depending on your library by the `expo` package with the version of core that is compatible with the specific SDK used in the project.<br/>
```

Comment:
```suggestion
Add the `expo` package as a peer dependency in your **package.json** — we recommend using `*` for the version range so as not to cause any duplicated packages in the user's **node_modules** folder. Your library also depends on `expo-modules-core` as a dev dependency — it's already provided by the `expo` package with a core version that is compatible with the specific SDK used in your project.<br/>
```

Reply:
To be honest I think `Your library also needs to depend on` is correct — it doesn't depend on `expo-modules-core` until you add it as a dev dependency 😅

Reply:
😄 Well then, can we explicitly say "`expo-modules-core` but only if it is added as a dev dependency"?

>it's already provided by the `expo` package with a core version that is compatible with the specific SDK used in your project

The above seems to imply that it is being added automatically.